src/docs/device_readme.md

```diff
@@ -1,4 +1,20 @@
-# Usage
-Make sure iotedge is installed on this device.  [[TODO: Link iotedge install docs here]]
+# Prerequisites
+Each device must have IoT Edge (must be v1.2 or later) installed. Pick the [supported OS](https://docs.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11) and follow the [tutorial](https://docs.microsoft.com/en-us/azure/iot-edge/support?view=iotedge-2020-11) to install Azure IoT Edge.
 
-From this folder, run `./install.sh`. This will copy the certificated, configure iot edge, and restart iotedge.
+Alternatively, if the OS is Tier1 supported OS and using DPS SAS Key provisioning, run [this installation script](https://github.com/Azure/iot-edge-config) can reduce the steps to install and provision.

```

Comment:
same comment as root: DPS SAS provisioning isn't supported rn

Reply:
Removed the texts completely